uint8 servo_id
uint8 register_address
uint8 bytes_number
int32 value

